server/channels/app/platform/log.go

```diff
@@ -67,12 +49,22 @@ func (ps *PlatformService) initLogging() error {
 		}
 	}
 
-	// Redirect default Go logger to app logger.
+	// redirect default Go logger to app logger.
 	ps.logger.RedirectStdLog(mlog.LvlStdLog)
 
-	// Use the app logger as the global logger (eventually remove all instances of global logging).
+	// use the app logger as the global logger (eventually remove all instances of global logging).
 	mlog.InitGlobalLogger(ps.logger)
 
+	// create notification logger if needed
+	if ps.notificationsLogger == nil {
```

Comment:
This is moved to after the main logger if configured so that it can log an errors in notification logging setup.